Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -49,6 +49,7 @@ public final class IndexStoreDB {
     useExplicitOutputUnits: Bool = false,
     waitUntilDoneInitializing wait: Bool = false,
     readonly: Bool = false,
+    enableOutOfDateFileWatching: Bool = false,
```

Comment:
Please add the new parameter to the doc comment.

Reply:
Done.